test/samples/local/foundry_expectRevert_nested/contracts/test/expectRevert.t.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+contract Test_expectRevertNested is Test {
+    event AssertionFailed(string message);
+
+    error E1();
+    error E2(string m);
+
+    uint256 private c;
+
+    function reverting() external returns (uint256) {
+        require(msg.sender == address(this));
+        revert E1();
+        return 1;
+    }
+
+    function callRevertingNoRevert() external returns (uint256) {
+	try this.reverting() {
+        	assert(false);
+	} catch {
+		return 1;
+	}
+    }
+
+    function test_expectRevertNested_0() external returns (uint256) {
+        vm.expectRevert(); // Expected to fail
+        try this.callRevertingNoRevert() returns (uint256 r) {
+            return 3;
+        } catch {
+            assert(false); //should get here
+        }
+    }
+
+    function notReverting() external returns (uint256) {
+        require(msg.sender == address(this));
+        return 1;
+    }
+
+    function callNotRevertingReverts() external returns (uint256) {
+	try this.notReverting() {
+        	assert(false);
+	} catch {
+		return 1;
+	}
+    }
+
+    function test_expectRevertNested_1() external returns (uint256) {
+        vm.expectRevert();
+        try this.callNotRevertingReverts() returns (uint256 r) {
+            assert(false); //should get here
+        } catch {
+            return 3;//shouldnt get here
```

Comment:
```solidity
//should get here
//shouldnt get here
```
Are pretty easy to confuse due to indifference in just two chars. Can we use only one of them (one or another)? Same is related to multiple other places.

Reply:
Good call, unfortunately fixing this would require me to re-run the entire test and fix 20-30 transaction JSONs manually... sooo feeling a little lazy on this one. Going to mark as not fix :)